src/frontend/apps/hub/src/styles/globals.scss

```
@@ -10,9 +10,17 @@
 @use './../features/chat/components/ChatBubble.scss';
 @use './../features/chat/components/ChatComposer.scss';
 @use './../features/chat/components/ChatConversation.scss';
+@use './../features/chat/components/ChatConversationSkeleton.scss';
```

Stylelint (17.12.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)